ngraph/core/src/op/or.cpp

```diff
@@ -22,7 +22,7 @@
 using namespace std;
 using namespace ngraph;
 
-constexpr NodeTypeInfo op::v1::LogicalOr::type_info;
+NGRAPH_RTTI_DEFINITION(op::v1::LogicalOr, "LogicalOr", 1, util::BinaryElementwiseLogical);
```

Comment:
Why is `LogicalOr` derived from `BinaryElementwiseLogical`, but other binary operations are not derived from corresponding base classes?